README.md

```diff
@@ -121,6 +121,14 @@ Note that Chrome will not allow a service worker to be registered over HTTPS wit
 
 prpl-server sets the [`Service-Worker-Allowed`](https://www.w3.org/TR/service-workers-1/#service-worker-allowed) header to `/` for any request path ending with `service-worker.js`. This allows a service worker served from a build subdirectory to be registered with a scope outside of that directory, e.g. `register('service-worker.js', {scope: '/'})`.
 
+## HTTPS
+
+Your apps should always be served over HTTPS. It protects your user's data, and is *required* for features like service workers and HTTP/2.
+
+If the `--https-redirect` flag is set, prpl-server will redirect all HTTP requests to HTTPS. It sends a `301 Moved Permanently` redirect to an `https://` address with the same hostname on the default HTTPS port (443). prpl-server trusts `X-Forwarded-*` headers from your reverse proxy to determine the client's true protocol and hostname.
```

Comment:
I keep wanting to hint at a potential scenario to help troubleshoot, should it happen that their proxy configuration didn't use X-Forwarded by default for some reason: 
> prpl-server relies on `X-Forwarded-*` headers from your reverse proxy to determine the client's true protocol and hostname (and to avoid infinite redirect loop!)